Source/Tenancy/Tenancy.csproj

```diff
@@ -1,16 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../../default.props"/>
+    <Import Project="../../default.props" />
 
-  <PropertyGroup>
-    <AssemblyName>Dolittle.Runtime.Tenancy</AssemblyName>
-  </PropertyGroup>
-  
-  <ItemGroup>
-    <ProjectReference Include="../ApplicationModel/ApplicationModel.csproj" />
-    <ProjectReference Include="../Configuration/Configuration.csproj" />
-    <ProjectReference Include="../Execution/Execution.csproj" />
-    <ProjectReference Include="../Types/Types.csproj" />
-    <ProjectReference Include="../Serialization.Json/Serialization.Json.csproj" />
-  </ItemGroup>
+    <PropertyGroup>
+        <AssemblyName>Dolittle.Runtime.Tenancy</AssemblyName>
+    </PropertyGroup>
+
+
+    <ItemGroup>
+        <PackageReference Include="Dolittle.Contracts" Version="$(ContractsVersion)" />
+        <PackageReference Include="Dolittle.Runtime.Contracts" Version="$(ContractsVersion)" />
+        <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsVersion)" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="../ApplicationModel/ApplicationModel.csproj" />
+        <ProjectReference Include="../Configuration/Configuration.csproj" />
+        <ProjectReference Include="../Execution/Execution.csproj" />
+        <ProjectReference Include="../Types/Types.csproj" />
+        <ProjectReference Include="../Serialization.Json/Serialization.Json.csproj" />
+        <ProjectReference Include="..\Services\Services.csproj" />
```

Comment:
Fix slash direction.

Reply:
Rider does that automatically. Would be nice if we could edit that in Rider

Reply:
Done